wifiphisher.py

```diff
@@ -875,10 +877,29 @@ def get_hostapd():
     # Only for systems with airmon-ng installed
     if os.path.isfile('/usr/sbin/airmon-ng'):
         proc = Popen(['airmon-ng', 'check', 'kill'], stdout=PIPE, stderr=DN)
-        
+
     # Get hostapd if needed
     get_hostapd()
 
+    # if template is incomplete locally, delete and ask for a download
+    if not phishingpage.check_template(TEMPLATE_NAME):
```

Comment:
That's fine, I will use a if condition and ```return True``` for those three templates. Would that be ok?